### ISSUE TITLE
SEA

### ISSUE THREAD

**Issue by GiraffeKey** [ArtemGr/gun#26]
The SEA API and User API which is built off SEA are arguably the most important features of GUN and the minimum we need for an MVP.

The first step is to implement everything seen here: https://gun.eco/docs/SEA

Note that certify is a new experimental feature that was added to the JS implementation recently; it's likely best to leave that one as a separate issue.

Once that is done we can build off it to implement the entire User API: https://gun.eco/docs/User

There is a lot to cover but I think with proper guidance we may be able to blaze through it.
